core/issuers.js

```diff
@@ -11,10 +11,40 @@ function makeMint(
   description,
   makeMintController = makeBasicMintController,
   makeAssay = makeNatAssay,
+  makeUseObj = () => {},
+  parent,
 ) {
   insist(description)`\
 Description must be truthy: ${description}`;
 
+  const parentIssuer = parent;
+  let childIssuer;
+  let childMint;
+
+  function getOrMakeChildMint() {
+    if (childMint === undefined) {
+      childMint = makeMint(
+        description,
+        makeMintController,
+        makeAssay,
+        makeUseObj,
+        issuer,
+      );
+      childIssuer = childMint.getIssuer();
+    }
+    return childMint;
+  }
+
+  function getChildAmount(parentAmount) {
+    getOrMakeChildMint();
+    // quantity is the same, but amounts are different for
+    // different issuers
+    const quantity = assay.quantity(parentAmount);
+    const childAssay = childIssuer.getAssay();
+    const childAmount = childAssay.make(quantity);
+    return childAmount;
+  }
+
   // src is a purse or payment. Return a fresh payment.  One internal
   // function used for both cases, since they are so similar.
   function takePayment(amount, isPurse, src, _name) {
```

Comment:
I don't see `isPurse` used below. This indicates two problems:
   * Why didn't our linting rules flag that we needed to rename the parameter `_isPurse`?
   * Doesn't our non-use mean that we might accidentally accept a payment where we were expecting only a purse, or vice versa?

Reply:
I'm not sure about the linting rules and why they didn't catch it. 
I think given that useRights and XferRights were taken out, we don't need to distinguish between purses and payments in this function anymore. https://github.com/Agoric/ERTP/pull/41/ takes isPurse out entirely